data/com/mojang/blaze3d/platform/NativeImage.mapping

```diff
@@ -11,13 +24,17 @@ CLASS com/mojang/blaze3d/platform/NativeImage
 		ARG 9 clamp
 		ARG 10 mipmap
 		ARG 11 autoClose
+	METHOD blendPixel (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 argb
```

Comment:
`otherPixelColor` implies that the color comes from another pixel, which while it may be true in nearly all cases, would add an unnecessary specificity as to the origin of the color, when `otherColor` or other names can suffice.

My opinion is that it be changed to `argbColor` (or perhaps renamed to `abgrColor`, as looking at the color unpacking helpers seems to imply that the actual order is `0xAABBGGRR`), which compromises between describing the value as a color and the format of the color.